draft-ietf-gnap-core-protocol.md

```diff
@@ -1431,7 +1431,8 @@ its capabilities and what is allowed to fulfill the request.
 
 In this non-normative example, the client instance is indicating that it can [redirect](#request-interact-redirect)
 the end user to an arbitrary URI and can receive a [redirect](#request-interact-callback-redirect) through
-a browser request.
+a browser request. Note that the client instance does not accept a push-style callback. This pattern
+is common for web-based client software.
```

Comment:
I would clarify "This redirect pattern" (to make sure we don't misunderstand with the note on push style callback that comes just before)